lib/licensee/project_files/license_file.rb

```diff
@@ -20,6 +20,11 @@ def self.name_score(filename)
         return 0.5 if filename =~ /licen[sc]e/i
         0.0
       end
+
+      # case-insensitive block to determine if the given file is LICENSE.lesser
+      def self.lesser_gpl_score(filename)
+        filename.casecmp('license.lesser').zero? ? 1 : 0
```

Comment:
Valid point. Although I'm not a fan of their implementation, I'm fine supporting LGPL's recommended means of manifesting your intent to license under LGPL. I'm less keen to support non-standard implementations, for the same reason we don't support `MIT.txt` or `LEGAL-RIGHTS.dog`. We can use proper detection as a carrot to motivate standardization.